pkg/endpoint/log.go

```diff
@@ -72,5 +75,8 @@ func (e *Endpoint) updateLogger() {
 		logfields.EndpointID:     e.ID,
 		logfields.ContainerID:    containerID,
 		logfields.PolicyRevision: e.policyRevision,
+		logfields.IPv4:           e.IPv4.String(),
+		logfields.IPv6:           e.IPv6.String(),
+		logfields.K8sPodName:     e.k8sPodName,
```

Comment:
This won't include the namespace. We have getters somewhere, but they might take ep.Mutex and deadlock :/

Reply:
Very good point, fixed

Reply:
We have `func (e *Endpoint) GetK8sNamespaceAndPodNameLocked() string {`